gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -36,6 +35,11 @@
         </dependencies>
     </dependencyManagement>
     <dependencies>
+        <dependency>
+            <groupId>uk.gov.gchq.gaffer</groupId>
+            <artifactId>gaas-common</artifactId>
+            <version>${project.parent.version}</version>
+        </dependency>
         <dependency>
             <groupId>io.springfox</groupId>
             <artifactId>springfox-boot-starter</artifactId>
```

Comment:
If `io.springfox` is in your dependency management do you need to provide the version?

Reply:
Removed version from the dependency so now its only in dependency management